vnext/Microsoft.ReactNative/ReactApplication.cpp

```diff
@@ -40,12 +41,15 @@ ReactApplication::ReactApplication(IInspectable const &outer) noexcept : ReactAp
 #ifndef USE_WINUI3
   Suspending({this, &ReactApplication::OnSuspending});
 
-#if defined _DEBUG && !defined DISABLE_XAML_GENERATED_BREAK_ON_UNHANDLED_EXCEPTION
+#if !defined DISABLE_XAML_GENERATED_BREAK_ON_UNHANDLED_EXCEPTION
   UnhandledException([this](IInspectable const &, UnhandledExceptionEventArgs const &e) {
```

Comment:
note that `UnhandledException` only gets triggered in _specific cases_ (@cglein can tell you all about how "easy" this is 😈). For example it won't be triggered if the exception happened in an async call in UWP... or maybe that was for win32?

Reply:
I found very little and conflicting information about SetUnhandledExceptionFilter vs. CoreApplication::UnhandledException . I suspect SetUnhandledExceptionFilter actually takes care of most native exception types; however, if there is managed code in the application I've found comments online saying unhandled exceptions in managed code don't actually trigger SetUnhandledExceptionFilter . but they do trigger CoreApplication::UnhandledException.
So really this is mostly optional for catching a few additional types of exceptions and routing them to the handler.

Reply:
[https://www.xsprogram.com/content/how-does-setunhandledexceptionfilter-work-in-net-winforms-applications.html](https://www.xsprogram.com/content/how-does-setunhandledexceptionfilter-work-in-net-winforms-applications.html)